### ISSUE TITLE
Siteimprove - Potential Issue - Mark banner videos as decorative

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#9168]
# Problem

Similar to the decorative images checkbox that was recently added to images, SiteImprove is flagging banner videos as potential issues for not having captions, etc.

## Flagged potential issues:
- Video missing captions
- Video without audio missing an accessible alternative

# Proposed Solution

Previous attempts and research indicate that the solution is likely going to be adding an `aria-hidden` attribute on the video element.

Do the above in a way that doesn't trigger a separate "Hidden element has focusable content". Previously, this might have happened because `aria-hidden` was applied to the Media entity wrapper div rather than the `<video>` element. It might have also been because there is a visible video control on banner images.

When this is done, update this draft article (or delete it and create a new one): https://webcommunity.sites.uiowa.edu/banner-video-flagged-missing-captions-or-accessible-alternatives

Revert #9577.

Optional: Add a description field to the video to allow users to provide a description that would be displayed appropriately as the text alternative and would suppress the `aria-hidden` behavior.

**Comment by bspeare** [comment]
@pyrello here's the process to mark it as a false positive for review: https://help.siteimprove.com/support/solutions/articles/80000962686-accessibility-how-to-dismiss-occurrences-of-accessibility-issues.

**Comment by pyrello** [comment]
Siteimprove responded and helped clarify the potential issue resolution workflow. I tested it out to understand how it works. It's a bit convoluted and the implication is that its going to require a bunch of site owners individually run through the process for their individual accounts. Not ideal.

It also appears there is now a second "potential issue" related to the banner block.
